<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> publications | Crombie Lab </title> <meta name="author" content=" "> <meta name="description" content="publications in reverse chronological order."> <meta name="keywords" content="Crombie Lab, Caenorhabditis elegans, academic-website, elegans, briggsae, tropicalis, caenorhabditis, quantitative genetics, genomics"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://crombie-lab.github.io/publications/"> <link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js?0afe9f0ae161375728f7bcc5eb5b4ab4"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> Crombie Lab </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about </a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/projects/">projects </a> </li> <li class="nav-item "> <a class="nav-link" href="/repositories/">repositories </a> </li> <li class="nav-item "> <a class="nav-link" href="/teaching/">teaching </a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fa-solid fa-moon"></i> <i class="fa-solid fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications</h1> <p class="post-description">publications in reverse chronological order.</p> </header> <article> <div class="publications"> <h2 class="bibliography">2023</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Shaver2023-hr" class="col-sm-8"> <div class="title">Variation in anthelmintic responses are driven by genetic differences among diverse C. elegans wild strains</div> <div class="author"> Amanda O Shaver ,  Janneke Wit ,  Clayton M Dilks , and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Timothy A Crombie, Hanchen Li, Raffi V Aroian, Erik C Andersen' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>PLoS Pathog.</em>, Apr 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Treatment of parasitic nematode infections in humans and livestock relies on a limited arsenal of anthelmintic drugs that have historically reduced parasite burdens. However, anthelmintic resistance (AR) is increasing, and little is known about the molecular and genetic causes of resistance for most drugs. The free-living roundworm Caenorhabditis elegans has proven to be a tractable model to understand AR, where studies have led to the identification of molecular targets of all major anthelmintic drug classes. Here, we used genetically diverse C. elegans strains to perform dose-response analyses across 26 anthelmintic drugs that represent the three major anthelmintic drug classes (benzimidazoles, macrocyclic lactones, and nicotinic acetylcholine receptor agonists) in addition to seven other anthelmintic classes. First, we found that C. elegans strains displayed similar anthelmintic responses within drug classes and significant variation across drug classes. Next, we compared the effective concentration estimates to induce a 10% maximal response (EC10) and slope estimates of each dose-response curve of each strain to the laboratory reference strain, which enabled the identification of anthelmintics with population-wide differences to understand how genetics contribute to AR. Because genetically diverse strains displayed differential susceptibilities within and across anthelmintics, we show that C. elegans is a useful model for screening potential nematicides before applications to helminths. Third, we quantified the levels of anthelmintic response variation caused by genetic differences among individuals (heritability) to each drug and observed a significant correlation between exposure closest to the EC10 and the exposure that exhibited the most heritable responses. These results suggest drugs to prioritize in genome-wide association studies, which will enable the identification of AR genes.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Crombie2023-ql" class="col-sm-8"> <div class="title">CaeNDR, the Caenorhabditis Natural Diversity Resource</div> <div class="author"> Timothy A Crombie ,  Ryan McKeown ,  Nicolas D Moya , and <span class="more-authors" title="click to view 12 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '12 more authors' ? 'Kathryn S Evans, Samuel J Widmayer, Vincent LaGrassa, Natalie Roman, Orzu Tursunova, Gaotian Zhang, Sophia B Gibson, Claire M Buchanan, Nicole M Roberto, Rodolfo Vieira, Robyn E Tanny, Erik C Andersen' : '12 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">12 more authors</span> </div> <div class="periodical"> <em>Nucleic Acids Res.</em>, Oct 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Studies of model organisms have provided important insights into how natural genetic differences shape trait variation. These discoveries are driven by the growing availability of genomes and the expansive experimental toolkits afforded to researchers using these species. For example, Caenorhabditis elegans is increasingly being used to identify and measure the effects of natural genetic variants on traits using quantitative genetics. Since 2016, the C. elegans Natural Diversity Resource (CeNDR) has facilitated many of these studies by providing an archive of wild strains, genome-wide sequence and variant data for each strain, and a genome-wide association (GWA) mapping portal for the C. elegans community. Here, we present an updated platform, the Caenorhabditis Natural Diversity Resource (CaeNDR), that enables quantitative genetics and genomics studies across the three Caenorhabditis species: C. elegans, C. briggsae and C. tropicalis. The CaeNDR platform hosts several databases that are continually updated by the addition of new strains, whole-genome sequence data and annotated variants. Additionally, CaeNDR provides new interactive tools to explore natural variation and enable GWA mappings. All CaeNDR data and tools are accessible through a freely available web portal located at caendr.org.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2022</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Crombie2022-ex" class="col-sm-8"> <div class="title">Local adaptation and spatiotemporal patterns of genetic diversity revealed by repeated sampling of Caenorhabditis elegans across the Hawaiian Islands</div> <div class="author"> Timothy A Crombie ,  Paul Battlay ,  Robyn E Tanny , and <span class="more-authors" title="click to view 12 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '12 more authors' ? 'Kathryn S Evans, Claire M Buchanan, Daniel E Cook, Clayton M Dilks, Loraina A Stinson, Stefan Zdraljevic, Gaotian Zhang, Nicole M Roberto, Daehan Lee, Michael Ailion, Kathryn A Hodgins, Erik C Andersen' : '12 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">12 more authors</span> </div> <div class="periodical"> <em>Mol. Ecol.</em>, Apr 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The nematode Caenorhabditis elegans is among the most widely studied organisms, but relatively little is known about its natural ecology. Genetic diversity is low across much of the globe but high in the Hawaiian Islands and across the Pacific Rim. To characterize the niche and genetic diversity of C. elegans on the Hawaiian Islands and to explore how genetic diversity might be influenced by local adaptation, we repeatedly sampled nematodes over a three-year period, measured various environmental parameters at each sampling site, and whole-genome sequenced the C. elegans isolates that we identified. We found that the typical Hawaiian C. elegans niche comprises moderately moist native forests at high elevations (500-1,500 m) where ambient air temperatures are cool (15-20°C). Compared to other Caenorhabditis species found on the Hawaiian Islands (e.g., Caenorhabditis briggsae and Caenorhabditis tropicalis), we found that C. elegans were enriched in native habitats. We measured levels of genetic diversity and differentiation among Hawaiian C. elegans and found evidence of seven genetically distinct groups distributed across the islands. Then, we scanned these genomes for signatures of local adaptation and identified 18 distinct regions that overlap with hyper-divergent regions, which may be maintained by balancing selection and are enriched for genes related to environmental sensing, xenobiotic detoxification, and pathogen resistance. These results provide strong evidence of local adaptation among Hawaiian C. elegans and contribute to our understanding of the forces that shape genetic diversity on the most remote volcanic archipelago in the world.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Crombie2022-ga" class="col-sm-8"> <div class="title">An automated approach to quantify chemotaxis index in C. elegans</div> <div class="author"> Timothy A Crombie ,  Chido Chikuturudzi ,  Daniel E Cook , and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Erik C Andersen' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>MicroPubl Biol</em>, May 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Chemotaxis assays are used extensively to study behavioral responses of Caenorhabditis nematodes to environmental cues. These assays result in a chemotaxis index (CI) that denotes the behavioral response of a population of nematodes to a particular compound and can range from 1 (maximum attraction) to -1 (maximum avoidance). Traditional chemotaxis assays have low throughput because researchers must manually setup experimental populations and score CIs. Here, we describe an automated methodology that increases throughput by using liquid-handling robots to setup experimental populations and a custom image analysis package, ct, to automate the scoring of CIs from plate images.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Crombie2022-zl" class="col-sm-8"> <div class="title">A Highly Scalable Approach to Perform Ecological Surveys of Selfing Caenorhabditis Nematodes</div> <div class="author"> Timothy A Crombie ,  Robyn E Tanny ,  Claire M Buchanan , and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Nicole M Roberto, Erik C Andersen' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>J. Vis. Exp.</em>, Mar 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Caenorhabditis elegans is one of the major model organisms in biology, but only recently have researchers focused on its natural ecology. The relative sparsity of information about C. elegans in its natural context comes from the challenges involved in the identification of the small nematode in nature. Despite these challenges, an increasing focus on the ecology of C. elegans has caused a wealth of new information regarding its life outside of the laboratory. The intensified search for C. elegans in nature has contributed to the discovery of many new Caenorhabditis species and revealed that congeneric nematodes frequently cohabitate in the wild, where they feed on microbial blooms associated with rotting plant material. The identification of new species has also revealed that the androdioecious mating system of males and self-fertilizing hermaphrodites has evolved three times independently within Caenorhabditis. The other two selfing species, C. briggsae and C. tropicalis, share the experimental advantages of C. elegans and have enabled comparative studies into the mechanistic basis of important traits, including self-fertilization. Despite these advances, much remains to be learned about the ecology and natural diversity of these important species. For example, we still lack functional information for many of their genes, which might only be attained through an understanding of their natural ecology. To facilitate ecological research of selfing Caenorhabditis nematodes, we developed a highly scalable method to collect nematodes from the wild. Our method makes use of mobile data collection platforms, cloud-based databases, and the R software environment to enhance researchers’ ability to collect nematodes from the wild, record associated ecological data, and identify wild nematodes using molecular barcodes.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Widmayer2022-ue" class="col-sm-8"> <div class="title">C. elegans toxicant responses vary among genetically diverse individuals</div> <div class="author"> Samuel J Widmayer ,  Timothy A Crombie ,  Joy N Nyaanga , and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Kathryn S Evans, Erik C Andersen' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>Toxicology</em>, Aug 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The genetic variability of toxicant responses among indisviduals in humans and mammalian models requires practically untenable sample sizes to create comprehensive chemical hazard risk evaluations. To address this need, tractable model systems enable reproducible and efficient experimental workflows to collect high-replication measurements of exposure cohorts. Caenorhabditis elegans is a premier toxicology model that has revolutionized our understanding of cellular responses to environmental pollutants and boasts robust genomic resources and high levels of genetic variation across the species. In this study, we performed dose-response analysis across 23 environmental toxicants using eight C. elegans strains representative of species-wide genetic diversity. We observed substantial variation in EC10 estimates and slope parameter estimates of dose-response curves of different strains, demonstrating that genetic background is a significant driver of differential toxicant susceptibility. We also showed that, across all toxicants, at least one C. elegans strain exhibited a significantly different EC10 or slope estimate compared to the reference strain, N2 (PD1074), indicating that population-wide differences among strains are necessary to understand responses to toxicants. Moreover, we quantified the heritability of responses (phenotypic variance attributable to genetic differences between individuals) to each toxicant exposure and observed a correlation between the exposure closest to the species-agnostic EC10 estimate and the exposure that exhibited the most heritable response. At least 20% of the variance in susceptibility to at least one exposure level of each compound was explained by genetic differences among the eight C. elegans strains. Taken together, these results provide robust evidence that heritable genetic variation explains differential susceptibility across an array of environmental pollutants and that genetically diverse C. elegans strains should be deployed to aid high-throughput toxicological screening efforts.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Fox2022-qr" class="col-sm-8"> <div class="title">C. elegans as a model for inter-individual variation in metabolism</div> <div class="author"> Bennett W Fox ,  Olga Ponomarova ,  Yong-Uk Lee , and <span class="more-authors" title="click to view 14 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '14 more authors' ? 'Gaotian Zhang, Gabrielle E Giese, Melissa Walker, Nicole M Roberto, Huimin Na, Pedro R Rodrigues, Brian J Curtis, Aiden R Kolodziej, Timothy A Crombie, Stefan Zdraljevic, L Safak Yilmaz, Erik C Andersen, Frank C Schroeder, Albertha J M Walhout' : '14 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">14 more authors</span> </div> <div class="periodical"> <em>Nature</em>, Jul 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Individuals can exhibit differences in metabolism that are caused by the interplay of genetic background, nutritional input, microbiota and other environmental factors1-4. It is difficult to connect differences in metabolism to genomic variation and derive underlying molecular mechanisms in humans, owing to differences in diet and lifestyle, among others. Here we use the nematode Caenorhabditis elegans as a model to study inter-individual variation in metabolism. By comparing three wild strains and the commonly used N2 laboratory strain, we find differences in the abundances of both known metabolites and those that have not to our knowledge been previously described. The latter metabolites include conjugates between 3-hydroxypropionate (3HP) and several amino acids (3HP-AAs), which are much higher in abundance in one of the wild strains. 3HP is an intermediate in the propionate shunt pathway, which is activated when flux through the canonical, vitamin-B12-dependent propionate breakdown pathway is perturbed5. We show that increased accumulation of 3HP-AAs is caused by genetic variation in HPHD-1, for which 3HP is a substrate. Our results suggest that the production of 3HP-AAs represents a ’shunt-within-a-shunt’ pathway to accommodate a reduction-of-function allele in hphd-1. This study provides a step towards the development of metabolic network models that capture individual-specific differences of metabolism and more closely represent the diversity that is found in entire species.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Crombie2022-bq" class="col-sm-8"> <div class="title">A Highly Scalable Approach to Perform Ecological Surveys of Selfing Caenorhabditis Nematodes</div> <div class="author"> Timothy A Crombie ,  Robyn E Tanny ,  Claire M Buchanan , and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Nicole M Roberto, Erik C Andersen' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>J. Vis. Exp.</em>, Mar 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Caenorhabditis elegans is one of the major model organisms in biology, but only recently have researchers focused on its natural ecology. The relative sparsity of information about C. elegans in its natural context comes from the challenges involved in the identification of the small nematode in nature. Despite these challenges, an increasing focus on the ecology of C. elegans has caused a wealth of new information regarding its life outside of the laboratory. The intensified search for C. elegans in nature has contributed to the discovery of many new Caenorhabditis species and revealed that congeneric nematodes frequently cohabitate in the wild, where they feed on microbial blooms associated with rotting plant material. The identification of new species has also revealed that the androdioecious mating system of males and self-fertilizing hermaphrodites has evolved three times independently within Caenorhabditis. The other two selfing species, C. briggsae and C. tropicalis, share the experimental advantages of C. elegans and have enabled comparative studies into the mechanistic basis of important traits, including self-fertilization. Despite these advances, much remains to be learned about the ecology and natural diversity of these important species. For example, we still lack functional information for many of their genes, which might only be attained through an understanding of their natural ecology. To facilitate ecological research of selfing Caenorhabditis nematodes, we developed a highly scalable method to collect nematodes from the wild. Our method makes use of mobile data collection platforms, cloud-based databases, and the R software environment to enhance researchers’ ability to collect nematodes from the wild, record associated ecological data, and identify wild nematodes using molecular barcodes.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Crombie2022-gl" class="col-sm-8"> <div class="title">Local adaptation and spatiotemporal patterns of genetic diversity revealed by repeated sampling of Caenorhabditis elegans across the Hawaiian Islands</div> <div class="author"> Timothy A Crombie ,  Paul Battlay ,  Robyn E Tanny , and <span class="more-authors" title="click to view 12 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '12 more authors' ? 'Kathryn S Evans, Claire M Buchanan, Daniel E Cook, Clayton M Dilks, Loraina A Stinson, Stefan Zdraljevic, Gaotian Zhang, Nicole M Roberto, Daehan Lee, Michael Ailion, Kathryn A Hodgins, Erik C Andersen' : '12 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">12 more authors</span> </div> <div class="periodical"> <em>Mol. Ecol.</em>, Apr 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The nematode Caenorhabditis elegans is among the most widely studied organisms, but relatively little is known about its natural ecology. Genetic diversity is low across much of the globe but high in the Hawaiian Islands and across the Pacific Rim. To characterize the niche and genetic diversity of C. elegans on the Hawaiian Islands and to explore how genetic diversity might be influenced by local adaptation, we repeatedly sampled nematodes over a three-year period, measured various environmental parameters at each sampling site, and whole-genome sequenced the C. elegans isolates that we identified. We found that the typical Hawaiian C. elegans niche comprises moderately moist native forests at high elevations (500-1,500 m) where ambient air temperatures are cool (15-20°C). Compared to other Caenorhabditis species found on the Hawaiian Islands (e.g., Caenorhabditis briggsae and Caenorhabditis tropicalis), we found that C. elegans were enriched in native habitats. We measured levels of genetic diversity and differentiation among Hawaiian C. elegans and found evidence of seven genetically distinct groups distributed across the islands. Then, we scanned these genomes for signatures of local adaptation and identified 18 distinct regions that overlap with hyper-divergent regions, which may be maintained by balancing selection and are enriched for genes related to environmental sensing, xenobiotic detoxification, and pathogen resistance. These results provide strong evidence of local adaptation among Hawaiian C. elegans and contribute to our understanding of the forces that shape genetic diversity on the most remote volcanic archipelago in the world.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2021</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Rajaei2021-ku" class="col-sm-8"> <div class="title">Mutability of mononucleotide repeats, not oxidative stress, explains the discrepancy between laboratory-accumulated mutations and the natural allele-frequency spectrum in C. elegans</div> <div class="author"> Moein Rajaei ,  Ayush Shekhar Saxena ,  Lindsay M Johnson , and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'Michael C Snyder, Timothy A Crombie, Robyn E Tanny, Erik C Andersen, Joanna Joyner-Matos, Charles F Baer' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>Genome Res.</em>, Sep 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Important clues about natural selection can be gleaned from discrepancies between the properties of segregating genetic variants and of mutations accumulated experimentally under minimal selection, provided the mutational process is the same in the laboratory as in nature. The base-substitution spectrum differs between C. elegans laboratory mutation accumulation (MA) experiments and the standing site-frequency spectrum, which has been argued to be in part owing to increased oxidative stress in the laboratory environment. Using genome sequence data from C. elegans MA lines carrying a mutation (mev-1) that increases the cellular titer of reactive oxygen species (ROS), leading to increased oxidative stress, we find the base-substitution spectrum is similar between mev-1, its wild-type progenitor (N2), and another set of MA lines derived from a different wild strain (PB306). Conversely, the rate of short insertions is greater in mev-1, consistent with studies in other organisms in which environmental stress increased the rate of insertion-deletion mutations. Further, the mutational properties of mononucleotide repeats in all strains are different from those of nonmononucleotide sequence, both for indels and base-substitutions, and whereas the nonmononucleotide spectra are fairly similar between MA lines and wild isolates, the mononucleotide spectra are very different, with a greater frequency of A:T →T:A transversions and an increased proportion of \pm1-bp indels. The discrepancy in mutational spectra between laboratory MA experiments and natural variation is likely owing to a consistent (but unknown) effect of the laboratory environment that manifests itself via different modes of mutability and/or repair at mononucleotide loci.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Di_Bernardo2021-bw" class="col-sm-8"> <div class="title">easyFulcrum: An R package to process and analyze ecological sampling data generated using the Fulcrum mobile application</div> <div class="author"> Matteo Di Bernardo ,  Timothy A Crombie ,  Daniel E Cook , and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Erik C Andersen' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>PLoS One</em>, Oct 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Large-scale ecological sampling can be difficult and costly, especially for organisms that are too small to be easily identified in a natural environment by eye. Typically, these microscopic floral and fauna are sampled by collecting substrates from nature and then separating organisms from substrates in the laboratory. In many cases, diverse organisms can be identified to the species-level using molecular barcodes. To facilitate large-scale ecological sampling of microscopic organisms, we used a geographic data-collection platform for mobile devices called Fulcrum that streamlines the organization of geospatial sampling data, substrate photographs, and environmental data at natural sampling sites. These sampling data are then linked to organism isolation data from the laboratory. Here, we describe the easyFulcrum R package, which can be used to clean, process, and visualize ecological field sampling and isolation data exported from the Fulcrum mobile application. We developed this package for wild nematode sampling, but it can be used with other organisms. The advantages of using Fulcrum combined with easyFulcrum are (1) the elimination of transcription errors by replacing manual data entry and/or spreadsheets with a mobile application, (2) the ability to clean, process, and visualize sampling data using a standardized set of functions in the R software environment, and (3) the ability to join disparate data to each other, including environmental data from the field and the molecularly defined identities of individual specimens isolated from samples.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Fouad2021-kw" class="col-sm-8"> <div class="title">High-throughput imaging of Caenorhabditis elegans aging using collective activity monitoring</div> <div class="author"> Anthony D Fouad ,  Matthew A Churgin ,  Julia Hayden , and <span class="more-authors" title="click to view 15 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '15 more authors' ? 'Joyce Xu, Jeong-Inn Park, Alice Liu, Christopher Teng, Hongjing Sun, Mateo Parrado, Peter Bowlin, Miguel De La Torre, Timothy A Crombie, Christine A Sedore, Anna L Coleman-Hulbert, Erik Johnson, Patrick Philips, Erik C Andersen, Christopher Fang-Yen' : '15 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">15 more authors</span> </div> <div class="periodical"> Oct 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The genetic manipulability and short lifespan of C. elegans make it an important model for aging research. Widely applied methods for measurements of worm aging based on manual observation are labor intensive and low-throughput. Here, we describe the Worm Collective Activity Monitoring Platform (WormCamp), a system for assaying aging in C. elegans by monitoring activity of populations of worms in standard 24-well plates. We show that metrics based on the rate of decline in collective activity can be used to estimate the average lifespan and locomotor healthspan in the population. Using the WormCamp, we assay a panel of highly divergent natural isolates of C. elegans and show that both lifespan and locomotor healthspan display substantial heritability. To facilitate analysis of large numbers of worms, we developed a robotic imaging system capable of simultaneous automated monitoring of activity, lifespan, and locomotor healthspan in up to 2,304 populations containing a total of  90,000 animals. We applied the automated system to conduct a large-scale RNA interference screen for genes that affect lifespan and locomotor healthspan. The WormCamp system is complementary to other current automated methods for assessing C. elegans aging and is well suited for efficiently screening large numbers of conditions. ### Competing Interest Statement The authors have declared no competing interest.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Nyaanga2021-aa" class="col-sm-8"> <div class="title">easyXpress: An R package to analyze and visualize high-throughput C. elegans microscopy data generated using CellProfiler</div> <div class="author"> Joy Nyaanga ,  Timothy A Crombie ,  Samuel J Widmayer , and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Erik C Andersen' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>PLoS One</em>, Aug 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>High-throughput imaging techniques have become widespread in many fields of biology. These powerful platforms generate large quantities of data that can be difficult to process and visualize efficiently using existing tools. We developed easyXpress to process and review C. elegans high-throughput microscopy data in the R environment. The package provides a logical workflow for the reading, analysis, and visualization of data generated using CellProfiler’s WormToolbox. We equipped easyXpress with powerful functions to customize the filtering of noise in data, specifically by identifying and removing objects that deviate from expected animal measurements. This flexibility in data filtering allows users to optimize their analysis pipeline to match their needs. In addition, easyXpress includes tools for generating detailed visualizations, allowing the user to interactively compare summary statistics across wells and plates with ease. Researchers studying C. elegans benefit from this streamlined and extensible package as it is complementary to CellProfiler and leverages the R environment to rapidly process and analyze large high-throughput imaging datasets.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Lee2021-qx" class="col-sm-8"> <div class="title">Balancing selection maintains hyper-divergent haplotypes in Caenorhabditis elegans</div> <div class="author"> Daehan Lee ,  Stefan Zdraljevic ,  Lewis Stevens , and <span class="more-authors" title="click to view 12 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '12 more authors' ? 'Ye Wang, Robyn E Tanny, Timothy A Crombie, Daniel E Cook, Amy K Webster, Rojin Chirakar, L Ryan Baugh, Mark G Sterken, Christian Braendle, Marie-Anne Félix, Matthew V Rockman, Erik C Andersen' : '12 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">12 more authors</span> </div> <div class="periodical"> <em>Nat Ecol Evol</em>, Apr 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Across diverse taxa, selfing species have evolved independently from outcrossing species thousands of times. The transition from outcrossing to selfing decreases the effective population size, effective recombination rate and heterozygosity within a species. These changes lead to a reduction in genetic diversity, and therefore adaptive potential, by intensifying the effects of random genetic drift and linked selection. Within the nematode genus Caenorhabditis, selfing has evolved at least three times, and all three species, including the model organism Caenorhabditis elegans, show substantially reduced genetic diversity relative to outcrossing species. Selfing and outcrossing Caenorhabditis species are often found in the same niches, but we still do not know how selfing species with limited genetic diversity can adapt to these environments. Here, we examine the whole-genome sequences from 609 wild C. elegans strains isolated worldwide and show that genetic variation is concentrated in punctuated hyper-divergent regions that cover 20% of the C. elegans reference genome. These regions are enriched in environmental response genes that mediate sensory perception, pathogen response and xenobiotic stress response. Population genomic evidence suggests that genetic diversity in these regions has been maintained by long-term balancing selection. Using long-read genome assemblies for 15 wild strains, we show that hyper-divergent haplotypes contain unique sets of genes and show levels of divergence comparable to levels found between Caenorhabditis species that diverged millions of years ago. These results provide an example of how species can avoid the evolutionary dead end associated with selfing.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2019</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Zdraljevic2019-mc" class="col-sm-8"> <div class="title">Natural variation in C. elegans arsenic toxicity is explained by differences in branched chain amino acid metabolism</div> <div class="author"> Stefan Zdraljevic ,  Bennett William Fox ,  Christine Strand , and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'Oishika Panda, Francisco J Tenjo, Shannon C Brady, Tim A Crombie, John G Doench, Frank C Schroeder, Erik C Andersen' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>Elife</em>, Apr 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>We find that variation in the dbt-1 gene underlies natural differences in Caenorhabditis elegans responses to the toxin arsenic. This gene encodes the E2 subunit of the branched-chain α-keto acid dehydrogenase (BCKDH) complex, a core component of branched-chain amino acid (BCAA) metabolism. We causally linked a non-synonymous variant in the conserved lipoyl domain of DBT-1 to differential arsenic responses. Using targeted metabolomics and chemical supplementation, we demonstrate that differences in responses to arsenic are caused by variation in iso-branched chain fatty acids. Additionally, we show that levels of branched chain fatty acids in human cells are perturbed by arsenic treatment. This finding has broad implications for arsenic toxicity and for arsenic-focused chemotherapeutics across human populations. Our study implicates the BCKDH complex and BCAA metabolism in arsenic responses, demonstrating the power of C. elegans natural genetic diversity to identify novel mechanisms by which environmental toxins affect organismal physiology. Editorial note: This article has been through an editorial process in which the authors decide how to respond to the issues raised during peer review. The Reviewing Editor’s assessment is that all the issues have been addressed (see decision letter).</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Crombie2019-os" class="col-sm-8"> <div class="title">Deep sampling of Hawaiian Caenorhabditis elegans reveals high genetic diversity and admixture with global populations</div> <div class="author"> Timothy A Crombie ,  Stefan Zdraljevic ,  Daniel E Cook , and <span class="more-authors" title="click to view 11 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '11 more authors' ? 'Robyn E Tanny, Shannon C Brady, Ye Wang, Kathryn S Evans, Steffen Hahnel, Daehan Lee, Briana C Rodriguez, Gaotian Zhang, Joost Zwagg, Karin Kiontke, Erik C Andersen' : '11 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">11 more authors</span> </div> <div class="periodical"> <em>Elife</em>, Dec 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Hawaiian isolates of the nematode species Caenorhabditis elegans have long been known to harbor genetic diversity greater than the rest of the worldwide population, but this observation was supported by only a small number of wild strains. To better characterize the niche and genetic diversity of Hawaiian C. elegans and other Caenorhabditis species, we sampled different substrates and niches across the Hawaiian islands. We identified hundreds of new Caenorhabditis strains from known species and a new species, Caenorhabditis oiwi. Hawaiian C. elegans are found in cooler climates at high elevations but are not associated with any specific substrate, as compared to other Caenorhabditis species. Surprisingly, admixture analysis revealed evidence of shared ancestry between some Hawaiian and non-Hawaiian C. elegans strains. We suggest that the deep diversity we observed in Hawaii might represent patterns of ancestral genetic diversity in the C. elegans species before human influence.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Zdraljevic2019-dt" class="col-sm-8"> <div class="title">Natural variation in C. elegans arsenic toxicity is explained by differences in branched chain amino acid metabolism</div> <div class="author"> Stefan Zdraljevic ,  Bennett William Fox ,  Christine Strand , and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'Oishika Panda, Francisco J Tenjo, Shannon C Brady, Timothy A Crombie, John G Doench, Frank C Schroeder, Erik C Andersen' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>Elife</em>, Apr 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>We find that variation in the dbt-1 gene underlies natural differences in Caenorhabditis elegans responses to the toxin arsenic. This gene encodes the E2 subunit of the branched-chain α-keto acid dehydrogenase (BCKDH) complex, a core component of branched-chain amino acid (BCAA) metabolism. We causally linked a non-synonymous variant in the conserved lipoyl domain of DBT-1 to differential arsenic responses. Using targeted metabolomics and chemical supplementation, we demonstrate that differences in responses to arsenic are caused by variation in iso-branched chain fatty acids. Additionally, we show that levels of branched chain fatty acids in human cells are perturbed by arsenic treatment. This finding has broad implications for arsenic toxicity and for arsenic-focused chemotherapeutics across human populations. Our study implicates the BCKDH complex and BCAA metabolism in arsenic responses, demonstrating the power of C. elegans natural genetic diversity to identify novel mechanisms by which environmental toxins affect organismal physiology. Editorial note: This article has been through an editorial process in which the authors decide how to respond to the issues raised during peer review. The Reviewing Editor’s assessment is that all the issues have been addressed (see decision letter).</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2018</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Crombie2018-rr" class="col-sm-8"> <div class="title">Head-to-head comparison of three experimental methods of quantifying competitive fitness in C. elegans</div> <div class="author"> Timothy A Crombie ,  Sayran Saber ,  Ayush Shekhar Saxena , and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Robyn Egan, Charles F Baer' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>PLoS One</em>, Oct 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Organismal fitness is relevant in many contexts in biology. The most meaningful experimental measure of fitness is competitive fitness, when two or more entities (e.g., genotypes) are allowed to compete directly. In theory, competitive fitness is simple to measure: an experimental population is initiated with the different types in known proportions and allowed to evolve under experimental conditions to a predefined endpoint. In practice, there are several obstacles to obtaining robust estimates of competitive fitness in multicellular organisms, the most pervasive of which is simply the time it takes to count many individuals of different types from many replicate populations. Methods by which counting can be automated in high throughput are desirable, but for automated methods to be useful, the bias and technical variance associated with the method must be (a) known, and (b) sufficiently small relative to other sources of bias and variance to make the effort worthwhile. The nematode Caenorhabditis elegans is an important model organism, and the fitness effects of genotype and environmental conditions are often of interest. We report a comparison of three experimental methods of quantifying competitive fitness, in which wild-type strains are competed against GFP-marked competitors under standard laboratory conditions. Population samples were split into three replicates and counted (1) “by eye” from a saved image, (2) from the same image using CellProfiler image analysis software, and (3) with a large particle flow cytometer (a “worm sorter”). From 720 replicate samples, neither the frequency of wild-type worms nor the among-sample variance differed significantly between the three methods. CellProfiler and the worm sorter provide at least a tenfold increase in sample handling speed with little (if any) bias or increase in variance.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Yeh2018-pa" class="col-sm-8"> <div class="title">The mutational decay of male-male and hermaphrodite-hermaphrodite competitive fitness in the androdioecious nematode C. elegans</div> <div class="author"> Shu-Dan Yeh ,  Ayush Shekhar Saxena ,  Timothy A Crombie , and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'Dorian Feistel, Lindsay M Johnson, Isabel Lam, Jennifer Lam, Sayran Saber, Charles F Baer' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>Heredity</em>, Jan 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Androdioecious Caenorhabditis have a high frequency of self-compatible hermaphrodites and a low frequency of males. The effects of mutations on male fitness are of interest for two reasons. First, when males are rare, selection on male-specific mutations is less efficient than in hermaphrodites. Second, males may present a larger mutational target than hermaphrodites because of the different ways in which fitness accrues in the two sexes. We report the first estimates of male-specific mutational effects in an androdioecious organism. The rate of male-specific inviable or sterile mutations is ⩽5 \times 10-4/generation, below the rate at which males would be lost solely due to those kinds of mutations. The rate of mutational decay of male competitive fitness is   0.17%/generation; that of hermaphrodite competitive fitness is   0.11%/generation. The point estimate of   1.5X faster rate of mutational decay of male fitness is nearly identical to the same ratio in Drosophila. Estimates of mutational variance (VM) for male mating success and competitive fitness are not significantly different from zero, whereas VM for hermaphrodite competitive fitness is similar to that of non-competitive fitness. Two independent estimates of the average selection coefficient against mutations affecting hermaphrodite competitive fitness agree to within two-fold, 0.33-0.5%.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Crombie2018-fr" class="col-sm-8"> <div class="title">Head-to-head comparison of three experimental methods of quantifying competitive fitness in C. elegans</div> <div class="author"> Timothy A Crombie ,  Sayran Saber ,  Ayush Shekhar Saxena , and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Robyn Egan, Charles F Baer' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>PLoS One</em>, Oct 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Organismal fitness is relevant in many contexts in biology. The most meaningful experimental measure of fitness is competitive fitness, when two or more entities (e.g., genotypes) are allowed to compete directly. In theory, competitive fitness is simple to measure: an experimental population is initiated with the different types in known proportions and allowed to evolve under experimental conditions to a predefined endpoint. In practice, there are several obstacles to obtaining robust estimates of competitive fitness in multicellular organisms, the most pervasive of which is simply the time it takes to count many individuals of different types from many replicate populations. Methods by which counting can be automated in high throughput are desirable, but for automated methods to be useful, the bias and technical variance associated with the method must be (a) known, and (b) sufficiently small relative to other sources of bias and variance to make the effort worthwhile. The nematode Caenorhabditis elegans is an important model organism, and the fitness effects of genotype and environmental conditions are often of interest. We report a comparison of three experimental methods of quantifying competitive fitness, in which wild-type strains are competed against GFP-marked competitors under standard laboratory conditions. Population samples were split into three replicates and counted (1) “by eye” from a saved image, (2) from the same image using CellProfiler image analysis software, and (3) with a large particle flow cytometer (a “worm sorter”). From 720 replicate samples, neither the frequency of wild-type worms nor the among-sample variance differed significantly between the three methods. CellProfiler and the worm sorter provide at least a tenfold increase in sample handling speed with little (if any) bias or increase in variance.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Crombie2018-xi" class="col-sm-8"> <div class="title">Head-to-Head Comparison of Three Methods of Quantifying Competitive Fitness in C. elegans</div> <div class="author"> Timothy A Crombie ,  Sayran Saber ,  Ayush Shekhar Saxena , and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Robyn Egan, Charles F Baer' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> Jul 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Organismal fitness is relevant in many contexts in biology. The most meaningful experimental measure of fitness is competitive fitness, when two or more entities (e.g., genotypes) are allowed to compete directly. In theory, competitive fitness is simple to measure: an experimental population is initiated with the different types in known proportions and allowed to evolve under experimental conditions to a predefined endpoint. In practice, there are several obstacles to obtaining robust estimates of competitive fitness in multicellular organisms, the most pervasive of which is simply the time it takes to count many individuals of different types from many replicate populations. Methods by which counting can be automated in high throughput are desirable, but for automated methods to be useful, the bias and technical variance associated with the method must be (a) known, and (b) sufficiently small relative to other sources of bias and variance to make the effort worthwhile. The nematode Caenorhabditis elegans is an important model organism, and the fitness effects of genotype and environmental conditions are often of interest. We report a comparison of three experimental methods of quantifying competitive fitness, in which wild-type strains are competed against GFP-marked competitors under standard laboratory conditions. Population samples were split into three replicates and counted (1) “by eye” from a saved image, (2) from the same image using CellProfiler image analysis software, and (3) with a large particle flow cytometer (a “worm sorter”). From 720 replicate samples, neither the frequency of wild-type worms nor the among-sample variance differed significantly between the three methods. CellProfiler and the worm sorter provide at least a tenfold increase in sample handling speed with little (if any) bias or increase in variance.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Yeh2018-nb" class="col-sm-8"> <div class="title">The mutational decay of male-male and hermaphrodite-hermaphrodite competitive fitness in the androdioecious nematode C. elegans</div> <div class="author"> Shu-Dan Yeh ,  Ayush Shekhar Saxena ,  Timothy A Crombie , and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'Dorian Feistel, Lindsay M Johnson, Isabel Lam, Jennifer Lam, Sayran Saber, Charles F Baer' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>Heredity</em>, Jan 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Androdioecious Caenorhabditis have a high frequency of self-compatible hermaphrodites and a low frequency of males. The effects of mutations on male fitness are of interest for two reasons. First, when males are rare, selection on male-specific mutations is less efficient than in hermaphrodites. Second, males may present a larger mutational target than hermaphrodites because of the different ways in which fitness accrues in the two sexes. We report the first estimates of male-specific mutational effects in an androdioecious organism. The rate of male-specific inviable or sterile mutations is ⩽5 \times 10-4/generation, below the rate at which males would be lost solely due to those kinds of mutations. The rate of mutational decay of male competitive fitness is   0.17%/generation; that of hermaphrodite competitive fitness is   0.11%/generation. The point estimate of   1.5X faster rate of mutational decay of male fitness is nearly identical to the same ratio in Drosophila. Estimates of mutational variance (VM) for male mating success and competitive fitness are not significantly different from zero, whereas VMfor hermaphrodite competitive fitness is similar to that of non-competitive fitness. Two independent estimates of the average selection coefficient against mutations affecting hermaphrodite competitive fitness agree to within two-fold, 0.33-0.5%.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2017</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Todgham2017-sk" class="col-sm-8"> <div class="title">The effect of temperature adaptation on the ubiquitin–proteasome pathway in notothenioid fishes</div> <div class="author"> A E Todgham ,  T A Crombie ,  and  others </div> <div class="periodical"> <em>Journal of Experimental</em>, Jan 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>There is an accumulating body of evidence suggesting that the sub-zero Antarctic marine environment places physiological constraints on protein homeostasis. Levels of ubiquitin (Ub)-conjugated proteins, 20S proteasome activity and mRNA expression of many proteins …</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Todgham2017-nz" class="col-sm-8"> <div class="title">The effect of temperature adaptation on the ubiquitin–proteasome pathway in notothenioid fishes</div> <div class="author"> Anne E Todgham ,  Timothy A Crombie ,  and  Gretchen E Hofmann </div> <div class="periodical"> <em>J. Exp. Biol.</em>, Feb 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>There is an accumulating body of evidence suggesting that the sub-zero Antarctic marine environment places physiological constraints on protein homeostasis. Levels of ubiquitin (Ub)-conjugated proteins, 20S proteasome activity and mRNA expression of many proteins involved in both the Ub tagging of damaged proteins as well as the different complexes of the 26S proteasome were measured to examine whether there is thermal compensation of the Ub–proteasome pathway in Antarctic fishes to better understand the efficiency of the protein degradation machinery in polar species. Both Antarctic ( Trematomus bernacchii , Pagothenia borchgrevinki ) and non-Antarctic ( Notothenia angustata , Bovichtus variegatus ) notothenioids were included in this study to investigate the mechanisms of cold adaptation of this pathway in polar species. Overall, there were significant differences in the levels of Ub-conjugated proteins between the Antarctic notothenioids and B. variegatus , with N. angustata possessing levels very similar to those of the Antarctic fishes. Proteasome activity in the gills of Antarctic fishes demonstrated a high degree of temperature compensation such that activity levels were similar to activities measured in their temperate relatives at ecologically relevant temperatures. A similar level of thermal compensation of proteasome activity was not present in the liver of two Antarctic fishes. Higher gill proteasome activity is likely due in part to higher cellular levels of proteins involved in the Ub–proteasome pathway, as evidenced by high mRNA expression of relevant genes. Reduced activity of the Ub–proteasome pathway does not appear to be the mechanism responsible for elevated levels of denatured proteins in Antarctic fishes, at least in the gills.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2016</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Crombie2016-ju" class="col-sm-8"> <div class="title">Inhibition of the oxidative stress response by heat stress in Caenorhabditis elegans</div> <div class="author"> Timothy A Crombie ,  Lanlan Tang ,  Keith P Choe , and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'David Julian' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>J. Exp. Biol.</em>, Jul 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>It has long been recognized that simultaneous exposure to heat stress and oxidative stress shows a synergistic interaction that reduces organismal fitness, but relatively little is known about the mechanisms underlying this interaction. We investigated the role of molecular stress responses in driving this synergistic interaction using the nematode Caenorhabditis elegans To induce oxidative stress, we used the pro-oxidant compounds acrylamide, paraquat and juglone. As expected, we found that heat stress and oxidative stress interact synergistically to reduce survival. Compared with exposure to each stressor alone, during simultaneous sublethal exposure to heat stress and oxidative stress the normal induction of key oxidative-stress response (OxSR) genes was generally inhibited, whereas the induction of key heat-shock response (HSR) genes was not. Genetically activating the SKN-1-dependent OxSR increased a marker for protein aggregation and decreased whole-worm survival during heat stress alone, with the latter being independent of HSF-1. In contrast, compared with wild-type worms, inactivating the HSR by HSF-1 knockdown, which would be expected to decrease basal heat shock protein expression, increased survival during oxidative stress alone. Taken together, these data suggest that, in C. elegans, the HSR and OxSR cannot be simultaneously activated to the same extent that each can be activated during a single stressor exposure. We conclude that the observed synergistic reduction in survival during combined exposure to heat stress and oxidative stress is due, at least in part, to inhibition of the OxSR during activation of the HSR.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Crombie2016-er" class="col-sm-8"> <div class="title">Inhibition of the oxidative stress response by heat stress in Caenorhabditis elegans</div> <div class="author"> Timothy A Crombie ,  Lanlan Tang ,  Keith P Choe , and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'David Julian' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>J. Exp. Biol.</em>, Jul 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>It has long been recognized that simultaneous exposure to heat stress and oxidative stress shows a synergistic interaction that reduces organismal fitness, but relatively little is known about the mechanisms underlying this interaction. We investigated the role of molecular stress responses in driving this synergistic interaction using the nematode Caenorhabditis elegans To induce oxidative stress, we used the pro-oxidant compounds acrylamide, paraquat and juglone. As expected, we found that heat stress and oxidative stress interact synergistically to reduce survival. Compared with exposure to each stressor alone, during simultaneous sublethal exposure to heat stress and oxidative stress the normal induction of key oxidative-stress response (OxSR) genes was generally inhibited, whereas the induction of key heat-shock response (HSR) genes was not. Genetically activating the SKN-1-dependent OxSR increased a marker for protein aggregation and decreased whole-worm survival during heat stress alone, with the latter being independent of HSF-1. In contrast, compared with wild-type worms, inactivating the HSR by HSF-1 knockdown, which would be expected to decrease basal heat shock protein expression, increased survival during oxidative stress alone. Taken together, these data suggest that, in C. elegans, the HSR and OxSR cannot be simultaneously activated to the same extent that each can be activated during a single stressor exposure. We conclude that the observed synergistic reduction in survival during combined exposure to heat stress and oxidative stress is due, at least in part, to inhibition of the OxSR during activation of the HSR.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2008</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Kabelik2008-ov" class="col-sm-8"> <div class="title">Aggression frequency and intensity, independent of testosterone levels, relate to neural activation within the dorsolateral subdivision of the ventromedial …</div> <div class="author"> D Kabelik ,  T Crombie ,  and  M C Moore </div> <div class="periodical"> <em>Horm. Behav.</em>, Jul 2008 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The mechanisms by which testosterone regulates aggression are unclear and may involve changes that alter the activity levels of one or more brain nuclei. We estimate neural activity by counting immunopositive cells against phosphorylated cyclic AMP response element …</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Kabelik2008-nk" class="col-sm-8"> <div class="title">Aggression frequency and intensity, independent of testosterone levels, relate to neural activation within the dorsolateral subdivision of the ventromedial hypothalamus in the tree lizard Urosaurus ornatus</div> <div class="author"> David Kabelik ,  Tim Crombie ,  and  Michael C Moore </div> <div class="periodical"> <em>Horm. Behav.</em>, Jun 2008 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The mechanisms by which testosterone regulates aggression are unclear and may involve changes that alter the activity levels of one or more brain nuclei. We estimate neural activity by counting immunopositive cells against phosphorylated cyclic AMP response element binding protein (pCREB). We demonstrate increased pCREB immunoreactivity within the dorsolateral subdivision of the ventromedial hypothalamus (VMHdl) following an aggressive encounter in male tree lizards Urosaurus ornatus. This immunoreactivity is induced both by exposure to and performance of aggressive behaviors. This dual activation of the VMHdl suggests its possible role as an integration center for assessment and expression of aggressive behavior. Furthermore, pCREB induction was greater in encounters involving higher frequency and intensity of aggressive display, demonstrating a direct relationship between neural activation and behavior. The VMHdl is also rich in steroid receptors. In a second experiment involving hormone manipulations, testosterone treatment increased aggression levels, though it did not increase the number of pCREB positive cells within the VMHdl. This lack of an effect of testosterone on pCREB induction within the VMHdl may be due to induction arising from the behaviors of conspecifics (especially in low-testosterone, low-aggression individuals), variation in aggression mediated by other variables, or regulation of aggression by circuits outside of the VMHdl. Together, these findings support a notion of the VMHdl as a nucleus involved in integrating afferent and efferent information within the neural aggression-control circuit.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2006</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Lehner2006-ph" class="col-sm-8"> <div class="title">Systematic mapping of genetic interactions in Caenorhabditis elegans identifies common modifiers of diverse signaling pathways</div> <div class="author"> Ben Lehner ,  Catriona Crombie ,  Julia Tischler , and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Angelo Fortunato, Andrew G Fraser' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>Nat. Genet.</em>, Aug 2006 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Most heritable traits, including disease susceptibility, are affected by interactions between multiple genes. However, we understand little about how genes interact because very few possible genetic interactions have been explored experimentally. We have used RNA interference in Caenorhabditis elegans to systematically test approximately 65,000 pairs of genes for their ability to interact genetically. We identify approximately 350 genetic interactions between genes functioning in signaling pathways that are mutated in human diseases, including components of the EGF/Ras, Notch and Wnt pathways. Most notably, we identify a class of highly connected ’hub’ genes: inactivation of these genes can enhance the phenotypic consequences of mutation of many different genes. These hub genes all encode chromatin regulators, and their activity as genetic hubs seems to be conserved across animals. We propose that these genes function as general buffers of genetic variation and that these hub genes may act as modifier genes in multiple, mechanistically unrelated genetic diseases in humans.</p> </div> </div> </div> </li></ol> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2024 . Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Last updated: January 28, 2024. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2930004b8d7fcd0a8e00fdcfc8fc9f24"></script> <script defer src="/assets/js/common.js?4a129fbf39254905f505c7246e641eaf"></script> <script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>